### PR TITLE
Update the UG on Get Student and Delete

### PR DESCRIPTION
**Pull request by jaysmyname**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -128,19 +128,55 @@ Examples:
...
-Deletes the specified person from the address book.
+Deletes the specified item from TACH.
 
-Format: `delete INDEX`
+#### Deleting Modules: `del -m`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+Format: `del -m [MODULE]…`
```
Comment (LowJiaHao99) [inline docs/UserGuide.md:137]: image

Are the brackets suppose to be there?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -128,19 +128,55 @@ Examples:
...
 
-Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* Deletes the modules listed.
+* **NOTE:** This action is **recursive**! It will also delete all classes and students assigned to the deleted modules.
+
+Example:
+* `del -m CS2103T` deletes the module and all of its classes and students like class `W15-3` in that module and
```
Comment (LowJiaHao99) [inline docs/UserGuide.md:143]: ... deletes the module and all of its classes and students. If class `W15-3` in is under `CS2103T` and student `Jack Smith` is under class `W-15-3`, both class `W15-3` and student `Jack Smith` will be deleted.

Does this sound clearer?

**Review by LowJiaHao99 — COMMENTED** [review]
Once resolved, it will be LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (jaysmyname) [reply docs/UserGuide.md:143]: Yeah it does sound clearer, I will edit it

File: docs/UserGuide.md (hunk 1 above)
Comment (jaysmyname) [reply docs/UserGuide.md:137]: I'll remove them

**Comment by LowJiaHao99** [comment]
LGTM

**Pull request merged**